### ISSUE TITLE
New IceBreakrrr details

### ISSUE THREAD

**Issue by drewe7192** [drewe7192/projectDate#101]

**Comment by drewe7192** [comment]
### Features:
- Live SpeedMeets
   - Route to screen and app immediately adds user to videoChat room and searches for match based on your preferences
   - cant find match after 10 secs? Screen pops up to try again or try other features
- Scheduled SpeedMeets
   - slot machine like game randomizes a group a potential matches based on your 
preferences/location
   - on click of "Spin button" 1 match is picked
   - user can decide if they want to either speedMeet right now or schedule for the future
      -  speedMeet right now sends message to that match requesting an immediate speedMeet
      - as soon as the match accepts a message/link is sent to both people to start videoChat (24 hour limit)
      - if user chooses scheduleSpeedMeet then they either can pick a time/ bind this speedMeet to a groupSpeedMeet   
- Group SpeedMeets
   - basically speedDating
   - maybe 3-4 matches
   - 1 minute each
   - host can either run this by themselves or add 1-2 more hosts
      - meaning the max video screens on mobile will  4
      - all hosts' screens will be on footer and horizontally scrollable
      - the guest will be big and centered on the screen 


### BottomTabs:
- Live SpeedMeets
- Match SpeedMeets (HomePage/slot machine)
- Current SpeedMeets (scheduled speedMeets)
- Discover SpeedMeets (other user's speedMeets)
